include/quicr/quicr_server.h

```diff
@@ -74,11 +74,11 @@ class ServerDelegate
    *         callbacks will be called. The delegate implementation
    *         shall decide the right callback for their usage.
    */
-  virtual void on_published_object(const QUICRName& quicr_name,
-                                   uint8_t priority,
-                                   uint64_t best_before,
-                                   bool use_reliable_transport,
-                                   bytes&& data) = 0;
+  virtual void onPublisherObject(const QUICRName& quicr_name,
+                                 uint8_t priority,
+                                 uint64_t best_before,
```

Comment:
I think this need to be in milii seconds after the message was recieved so it can be perhasp uint16_t